demo/index.html

```diff
@@ -84,48 +89,48 @@
       <div>
         <h4>Enabled</h4>
         <div class="horizontal-section">
-          <paper-icon-button icon="menu" title="menu"></paper-icon-button>
-          <paper-icon-button icon="favorite" title="heart"></paper-icon-button>
-          <paper-icon-button icon="arrow-back" title="arrow-back"></paper-icon-button>
-          <paper-icon-button icon="arrow-forward" title="arrow-forward"></paper-icon-button>
-          <paper-icon-button icon="clear" title="clear"></paper-icon-button>
-          <paper-icon-button icon="polymer" title="polymer"></paper-icon-button>
-          <paper-icon-button src="https://assets-cdn.github.com/images/modules/logos_page/Octocat.png" aria-label="octocat" title="octocat"></paper-icon-button>
+          <paper-icon-button icon="menu" title="menu"></paper-icon-button><br>
```

Comment:
Done! Changed back to `display:block` and set the right sizes (i wanted a round ripple instead of a width-of-the-whole-box ripple)